scalableminds/webknossos
# Brush presets

### Steps to test:
- Go into brush mode when viewing an annotation
- Open the change brushsize popover
- Make sure there are buttons you can click to set the brush size to small, medium and large
- The brush itself should be visible in the center of the active viewport

### Issues:
- partly fixes #7046: 
-- creates buttons with preset brush sizes
-- renders brush in the center of the active viewport
- fixes #5363

------
(Please delete unneeded items, merge only when none are left open)
